### ISSUE TITLE
terraform-docs broken in 1.11.0 ?

### ISSUE THREAD

**Issue by gregbook** [antonbabenko/pre-commit-terraform#39]
Hi, trying to use pre-commit hooks for terraform-docs but every-time the hook is passed no readme.md file is created. It works well for terraform-fmt though and modify the files accordingly. Thanks for your support!

**System Information:**
> MacOS : 10.14.4 (18E226)
Terraform-docs : 0.6.0
Bash version : GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin18)
Terraform v0.11.13
Python 3.7.3

**Files:**
```.
├── main.tf
├── output.tf
└── variable.tf
```

**.pre-commit-hook.yaml:**
```- repo: git://github.com/antonbabenko/pre-commit-terraform
  rev: v1.11.0
  hooks:
    - id: terraform_fmt
    - id: terraform_docs
```

**Issue:**
```
% pre-commit run -c .pre-commit-config.yaml --all-files --verbose
[terraform_fmt] Terraform fmt............................................Passed
[terraform_docs] Terraform docs..........................................Passed
```

**Comment by glerma** [comment]
Yah, it's not working for me either.  I'm on MacOS 10.14.4 as well.

I keep getting a"Failed", no matter what, and won't allow me to commit.

**Comment by glerma** [comment]
I can't get this to work on my MacOS 10.14.4 either.

**Comment by moxli** [comment]
```
It does not work for me either.

pre-commit run -a  
Terraform fmt............................................................Passed
Terraform docs...........................................................Failed
hookid: terraform_docs

/home/mdorner/.cache/pre-commit/repoq21wt094/terraform_docs.sh: line 65: terraform-docs: command not found

Terraform validate with variables........................................Passed

```
The problem seems to be that it tries to call a binary called 'terraform-docs' in line 65 which is not in $PATH.

`terraform-docs $args md ./ > "$tmp_file"`

**Comment by snemetz** [comment]
@moxli You need to install the dependencies and make sure they are on your path. This is not an issue with the hooks. But if you are on a Mac, my fork has a hook to install the dependencies with homebrew

**Comment by krzyzakp** [comment]
Sad to say, but terraform-docs won't work with 0.12. Of course if you upgrade your code properly to that version. Tested by running manually, it's failing with error: `2019/06/07 14:25:53 At 2:12: Unknown token: 2:12 IDENT var.region` - as in top scope we don't need to use `"${var.region}` anymore, just can and should use `var.region`.

Looking on that project, it seems to be abandon - last changes nearly half year ago, before there was something happening every month.

**Comment by antonbabenko** [comment]
terraform-docs and Terraform 0.12 support has been released in [v1.13.0](https://github.com/antonbabenko/pre-commit-terraform/releases/v1.13.0).
